src/Locations/Locations.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+
+export class Locations extends Component {
+    constructor(){
+        super()
+
+        this.state = {
+            hidden: true,
+            list: []
+        }
+    }
+
+    handleClick = () => {
+        this.setState({ hidden: !this.state.hidden })
+    }
+
+    getLocations = async () => {
+        const { data } = await axios.get(`https://ghibliapi.herokuapp.com/locations/`)
+        this.setState({ list: data.map((location) => location.name)})
```

Comment:
Nice map here! Makes the map in your render much clearer when you're only working with the names.